doc/OpenCV_docs/OpenCV_SVM_vs_Shogun_SVM.md

```diff
@@ -191,7 +192,7 @@ We start with creating the training data as the ```DenseFeatures```.
 
 ```CPP
     SGMatrix<float64_t> shogun_traindata = CV2SGFactory::get_sgmatrix<float64_t>(traindata);
```

Comment:
maybe we can change this method to transpose the matrix upon creation? rather than doing it afterwards?